src/components/button/Button.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import styles from "src/components/button/Button.module.scss";
+
+const Button = ({
+  buttonText,
+  iconSrc,
+  altText,
+  buttonClass,
+  isButtonDisabled,
+  onButtonClick,
+}) => {
+  const combinedClass = classNames(styles["button"], buttonClass);
```

Comment:
Is there any reason behind accessing style class with bracket notation ? May be using `dot(.)` notation will provide much cleaner look. [reference](https://github.com/css-modules/css-modules)
![Screenshot 2023-05-31 at 9 30 43 AM](https://github.com/rsa-wtag/Todos/assets/128278816/e43201a5-5be4-49cc-aedc-5ccd1cec8476)